db/database.sql

```diff
@@ -50,6 +50,7 @@ CREATE TABLE IF NOT EXISTS `file_data` (
   `data` blob NOT NULL,
   `timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
   `created_by` varchar(255) NOT NULL,
+  `comment` varchar(140) DEFAULT NULL,
```

Comment:
I didn't want them to be long explanations, and so I thought the length of a Tweet would be a good metric. :P

What other value seems better? 255?